bpf/cpu-profiler/src/main.rs

```diff
@@ -32,16 +32,17 @@ pub static mut STACK_TRACES: StackTrace = StackTrace::with_max_entries(MAX_STACK
 
 #[perf_event]
 pub fn profile_cpu(ctx: PerfEventContext) -> u32 {
-    match unsafe { try_profile_cpu(ctx) } {
-        Ok(ret) => ret,
-        Err(ret) => ret,
+    unsafe {
+        try_profile_cpu(ctx);
     }
+
+    0
 }
 
 #[inline(always)]
-unsafe fn try_profile_cpu(ctx: PerfEventContext) -> Result<u32, u32> {
+unsafe fn try_profile_cpu(ctx: PerfEventContext) {
     if ctx.pid() == 0 {
```

Comment:
Unrelated to this PR, but I noticed it now, we should not have this check as pid=0 is a special kernel process (swapper) and we might want to take it into account

```
$ sudo bpftrace -e 'profile:hz:99 /pid==0/ {@[kstack]=count()}'
```